cmd/cloud-controller-manager/nodeipamcontroller.go

```diff
@@ -79,14 +79,14 @@ func startNodeIpamController(ccmConfig *cloudcontrollerconfig.CompletedConfig, n
 	if len(strings.TrimSpace(nodeIPAMConfig.ServiceCIDR)) != 0 {
 		_, serviceCIDR, err = net.ParseCIDR(nodeIPAMConfig.ServiceCIDR)
 		if err != nil {
-			klog.Warningf("Unsuccessful parsing of service CIDR %v: %v", nodeIPAMConfig.ServiceCIDR, err)
+			klog.InfoS("Unsuccessful parsing of service CIDR %v: %v", nodeIPAMConfig.ServiceCIDR, err)
```

Comment:
Please take a look at https://kubernetes.io/blog/2020/09/04/kubernetes-1-19-introducing-structured-logs/. Switching to structured logs requires adding field names to the list of parameters.